### DISCUSSION TITLE
Incorrect detection of TS

### DISCUSSION THREAD

**Discussion by coreyfarrell** [npm/feedback#207] [Website] [answered]
Recently the website started indicating if a package has TypeScript declarations.  TypeScript is detected for [get-package-type](https://www.npmjs.com/package/get-package-type) but this package does not contain any TypeScript.

**Comment by MylesBorins** [comment]
/cc @ethomson 

I've opened an internal ticket to report to the team

**Comment by lumaxis** [comment, marked as answer]
@coreyfarrell I just deployed a bugfix for this issue. If a package was wrongly detected, this will be fixed on the next package publish.
